src/cpu.h

```diff
@@ -23,6 +24,7 @@ struct Memory {
   std::array<Byte, MEM_LIMIT> data;
 
   void initialise();
+  bool initialise(std::ifstream &input_file);
```

Comment:
Future plan: maybe there should be another function to write a long character string in memory ( or a constructor using initialiser list )